changelogs/unreleased/gh-9112-upgrade-drop-constraint.md

```diff
@@ -0,0 +1,6 @@
+## feature/sql
+
+* Introduced `DROP CONSTRAINT` variation to drop field constraints (gh-9136).
+* `DROP CONSTRAINT` is now prohibited if a given name matches more than one
+  constraint.
+* Introduced `DROP CONSTRAINT` variation for each constraint type.
```

Comment:
```suggestion
* Introduced a `DROP CONSTRAINT` variation to drop field constraints (gh-9136).
* `DROP CONSTRAINT` is now prohibited if a given name matches more than one
  constraint.
* Introduced a `DROP CONSTRAINT` variation for each constraint type.
```

Reply:
Thanks, fixed! Also, I corrected the issue number and added `(gh-9112)` at the end of each line. This is fine?

Reply:
@andreyaksenov I changed a changelog. Could you take a look?

Reply:
@ImeevMA looks good, thanks